.editorconfig

```diff
@@ -0,0 +1,18 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
here's where we'd decide on 2 vs 4. I prefer 4, but I'll go with whatever we settle on.

Reply:
I also prefer 4